src/pages/Florist/index.tsx

```diff
@@ -180,12 +180,16 @@ export const Florist = () => {
             </Typography>
           ) : (
             <Grid className={classes.subtitle}>
-              <Typography variant="h6">
-                Welcome to our store. 
-              </Typography>
-              <Typography variant="body1" className={classes.welcomeText}>
-                Why don't you take a look around?
-              </Typography>
+              <Typography variant="h6">Welcome to our store.</Typography>
+              {!userData.balance ? (
+                <Typography variant="body1" className={classes.welcomeText}>
+                  To purchase flowers, let grew up flower and get coin!
```

Comment:
The text could be something like "To purchase flowers you need coins! Plant some seeds and your wallet will grow."